deploy/k3s/edge_deployment/edge_deployment.yaml

```diff
@@ -75,6 +75,10 @@ spec:
         # TODO: Once we have kubernetes-based tests, we can remove this feature flag.
         - name: DEPLOY_DETECTOR_LEVEL_INFERENCE
           value: "1"
+        # When this is set to ENABLED, ask_async calls to the edge endpoint will only return the edge model's answer,
+        # regardless of the confidence.
+        - name: EDGE_ONLY
+          value: "" # disabled by default
```

Comment:
Can we use an actual value that can be converted to a boolean here like "0" instead of an empty string?

Reply:
Yeah for sure, should I also change the logic to interpret 0 / 1 values? Or keep it as is (where EDGE_ONLY must be "ENABLED").

Reply:
Bumping this question.

Reply:
Change the logic to interpret 0/1 values, like bool(int("1"))